ebos/eclfluxmodule.hh

```diff
@@ -278,10 +278,6 @@ public:
                                         J,
                                         distZ*g,
                                         thpres);
-            if (pressureDifferences[phaseIdx] == 0) {
-                volumeFlux[phaseIdx] = 0.0;
-                continue;
-            }
```

Comment:
I don't really understand why this changes anything. Souldn't `volumeFlux[phaseindex]` still be zero as there is a multiplication with zero below. What am I missing?